components/kyma-environment-broker/internal/broker/plans.go

```diff
@@ -209,34 +209,80 @@ func createSchemaWith(properties interface{}, update bool) *map[string]interface
 // keep internal/hyperscaler/azure/config.go in sync with any changes to available zones
 func Plans(plans PlansConfig, provider internal.CloudProvider, includeAdditionalParamsInSchema bool) map[string]domain.ServicePlan {
 	awsMachines := []string{"m5.xlarge", "m5.2xlarge", "m5.4xlarge", "m5.8xlarge", "m5.12xlarge", "m6i.xlarge", "m6i.2xlarge", "m6i.4xlarge", "m6i.8xlarge", "m6i.12xlarge"}
+	awsMachinesDisplay := map[string]string{
+		// source: https://aws.amazon.com/ec2/instance-types/m5/
+		"m5.xlarge":   "m5.xlarge (4vCPU, 16GB RAM)",
+		"m5.2xlarge":  "m5.2xlarge (8vCPU, 32GB RAM)",
+		"m5.4xlarge":  "m5.4xlarge (16vCPU, 64GB RAM)",
+		"m5.8xlarge":  "m5.8xlarge (32vCPU, 128GB RAM)",
+		"m5.12xlarge": "m5.12xlarge (48vCPU, 192GB RAM)",
+		// source: https://aws.amazon.com/ec2/instance-types/m6i/
+		"m6i.xlarge":   "m6i.xlarge (4vCPU, 16GB RAM)",
+		"m6i.2xlarge":  "m6i.2xlarge (8vCPU, 32GB RAM)",
+		"m6i.4xlarge":  "m6i.4xlarge (16vCPU, 64GB RAM)",
+		"m6i.8xlarge":  "m6i.8xlarge (32vCPU, 128GB RAM)",
+		"m6i.12xlarge": "m6i.12xlarge (48vCPU, 192GB RAM)",
+	}
 
-	// awsHASchema := AWSHASchema(awsMachines, includeAdditionalParamsInSchema, false)
+	// awsHASchema := AWSHASchema(awsMachinesDisplay, awsMachines, includeAdditionalParamsInSchema, false)
 
+	// source: https://cloud.google.com/compute/docs/general-purpose-machines#e2_limitations
 	gcpMachines := []string{"n2-standard-4", "n2-standard-8", "n2-standard-16", "n2-standard-32", "n2-standard-48"}
-	gcpSchema := GCPSchema(gcpMachines, includeAdditionalParamsInSchema, false)
+	gcpMachinesDisplay := map[string]string{
+		"n2-standard-4":  "n2-standard-4 (4vCPU, 16GB RAM)",
+		"n2-standard-8":  "n2-standard-8 (8vCPU, 32GB RAM)",
+		"n2-standard-16": "n2-standard-16 (16vCPU, 64GB RAM)",
+		"n2-standard-32": "n2-standard-32 (32vCPU, 128GB RAM)",
+		"n2-standard-48": "n2-standard-48 (48vCPU, 192B RAM)",
+	}
+	gcpSchema := GCPSchema(gcpMachinesDisplay, gcpMachines, includeAdditionalParamsInSchema, false)
 
 	openStackMachines := []string{"g_c4_m16", "g_c8_m32"}
-	openstackSchema := OpenStackSchema(openStackMachines, includeAdditionalParamsInSchema, false)
+	openStackMachinesDisplay := map[string]string{
+		"g_c4_m16": "g_c4_m16 (4vCPU, 16GB RAM)",
+		"g_c8_m32": "g_c8_m32 (8vCPU, 32GB RAM)",
```

Comment:
I couldn't find any docs regarding this, the issue as reported also doesn't specify anything regarding OpenStack so this is my guess what the values probably mean